src/command_modules/azure-cli-reservations/HISTORY.rst

```diff
@@ -3,6 +3,20 @@
 Release History
 ===============
 
+0.2.0
++++++
```

Comment:
Please add some line indicating these include breaking changes in CAPS or **Bold**
ex: https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-network/HISTORY.rst#011b1-2017-01-17

Reply:
done